__tests__/app.test.js

```diff
@@ -262,3 +262,69 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("status:200, should respond with an object containing the updated article after the votes property is incremented correctly", () => {
+    const articleUpdate = { inc_votes: 5 };
+
+    return request(app)
+      .patch("/api/articles/3")
+      .send(articleUpdate)
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toEqual({
+          article_id: 3,
+          title: "Eight pug gifs that remind me of mitch",
+          topic: "mitch",
+          author: "icellusedkars",
+          body: "some gifs",
+          created_at: expect.any(String),
+          votes: 7,
+        });
```

Comment:
Please refactor this test so it's not 'brittle' in future extensions of the data. E.g. if a client asks for article data to have a "isFavourited" property to article data then your current test will fail immediately. It's better to make this flexible using `toMatchObject` or `objectContaining` matchers instead of `toEqual`

Reply:
Understood!